examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -3,15 +3,14 @@
  * Licensed under the MIT License.
  */
 import { KeyValueDataObject } from "@fluid-experimental/data-objects";
-import Fluid from "@fluid-experimental/fluid-static";
-import { TinyliciousService } from "@fluid-experimental/get-container";
+import { FluidTinylicious } from "@fluid-experimental/fluid-static";
+import { ITinyliciousFileConfig } from "@fluid-experimental/get-container";
 import { SharedMap } from "@fluidframework/map";
 import { DiceRollerController } from "./controller";
 import { renderDiceRoller } from "./view";
 
 // Define the server we will be using and initialize Fluid
-const service = new TinyliciousService();
-Fluid.init(service);
+FluidTinylicious.init();
```

Comment:
If we keep the init I see three configs.

1. Service specific properties that apply across the application (ex. FRS endpoint, tokens)
2. Container specific properties that are required for the service (ex. ID)
3. Container specific properties that define the data schema

Reply:
Correct. There is an initial config to set up the service itself. Then there are two more for the container, one which is service-specific and the other defining the objects within the container